sats-dna/src/main/kotlin/com/sats/dna/components/button/SatsButton.kt

```diff
@@ -71,7 +71,9 @@ fun SatsButton(
         colors = buttonColors,
         contentPadding = buttonPadding(isLarge),
     ) {
-        Row(Modifier.height(24.dp), verticalAlignment = CenterVertically) {
```

Comment:
I seem to remember there was a reason we set this `height` specifically, but I can't figure out why now. Removing it obviously fixes the clipping, but if you can think of or remember why we had it there in the first place, then yell out.

Reply:
I have no idea, could it be that the button got too big in certain instances? Overlapping other content in a weird way if it was a hovering button. If that's the case it should definitely be solved in a different way than setting fixed height on all button text though

Reply:
Might have been because we had a `fillMaxHeight` in there that also didn't make much sense. I removed both of them. 😆